src/client/clang.cc

```diff
@@ -20,7 +20,8 @@ namespace client {
 bool DoMain(int argc, const char* const argv[], Immutable socket_path,
             Immutable clang_path, Immutable version, ui32 connect_timeout_secs,
             ui32 read_timeout_secs, ui32 send_timeout_secs, ui32 read_min_bytes,
-            const HashMap<String, String>& plugins, bool disabled) {
+            const HashMap<String, String>& plugins, bool disabled,
+            const bool rewrite_includes) {
```

Comment:
Why `const` for function argument here? At least it's not a common style in other parts of code.